Helper/Config.php

```diff
@@ -655,6 +670,15 @@ public function isSandboxModeSet($store = null)
         );
     }
 
+    public function getCustomURLValueOrDefault($path, $default)
+    {
+        $storedValue = $this->getScopeConfig()->getValue($path);
+        if ($storedValue == "") {
```

Comment:
return $this->getScopeConfig()->getValue($path) ?? $default

Reply:
can't use coalesce here, the string is empty not null which coalesce doesnt check.

i can use `?:` but that will throw an E_NOTICE if not set, maybe i should be more explicit

Reply:
i used `empty()` that should work.